Directory.Packages.props

```diff
@@ -80,35 +76,33 @@
     <PackageVersion Include="PocketLogger.For.ApplicationInsights" Version="0.8.0" />
     <PackageVersion Include="PocketLogger.For.Xunit" Version="0.9.0" />
     <PackageVersion Include="PocketLogger.Subscribe" Version="0.9.0" />
-    <PackageVersion Include="PocketLogger" Version="0.9.1" />
+    <PackageVersion Include="PocketLogger" Version="0.10.0" />
     <PackageVersion Include="PowerShellGet" Version="2.2.5" />
     <PackageVersion Include="Serilog.Sinks.RollingFileAlternate" Version="2.0.9" />
     <PackageVersion Include="Serilog" Version="4.2.0" />
-    <PackageVersion Include="SkiaSharp" Version="2.88.6" />
+    <PackageVersion Include="SkiaSharp" Version="3.119.0" />
     <PackageVersion Include="StreamJsonRpc" Version="2.21.69" />
-    <PackageVersion Include="System.Collections.Immutable" Version="9.0.5" />
+    <PackageVersion Include="System.Collections.Immutable" Version="9.0.6" />
     <PackageVersion Include="System.CommandLine.NamingConventionBinder" Version="2.0.0-beta4.22272.1" />
     <PackageVersion Include="System.CommandLine" Version="2.0.0-beta4.22272.1" />
     <PackageVersion Include="System.ComponentModel.Annotations" Version="5.0.0" />
-    <PackageVersion Include="System.Diagnostics.DiagnosticSource" Version="9.0.5" />
-    <PackageVersion Include="System.Drawing.Common" Version="9.0.5" />
-    <PackageVersion Include="System.IO.Pipelines" Version="9.0.5" />
+    <PackageVersion Include="System.Diagnostics.DiagnosticSource" Version="9.0.6" />
+    <PackageVersion Include="System.Drawing.Common" Version="9.0.6" />
+    <PackageVersion Include="System.IO.Pipelines" Version="9.0.6" />
     <PackageVersion Include="System.Linq.Async" Version="6.0.1" />
-    <PackageVersion Include="System.Memory" Version="4.6.0" />
+    <PackageVersion Include="System.Memory" Version="4.6.3" />
     <PackageVersion Include="System.Net.Http" Version="4.3.4" />
     <PackageVersion Include="System.Reactive" Version="6.0.0" />
-    <PackageVersion Include="System.Reflection.Metadata" Version="9.0.5" />
-    <PackageVersion Include="System.Runtime.Caching" Version="9.0.5" />
+    <PackageVersion Include="System.Reflection.Metadata" Version="9.0.6" />
+    <PackageVersion Include="System.Runtime.Caching" Version="9.0.6" />
     <PackageVersion Include="System.Runtime.InteropServices" Version="4.3.0" />
     <PackageVersion Include="System.Runtime.Loader" Version="4.3.0" />
-    <PackageVersion Include="System.Security.Cryptography.Pkcs" Version="9.0.5" />
+    <PackageVersion Include="System.Security.Cryptography.Pkcs" Version="9.0.6" />
     <PackageVersion Include="System.Security.Cryptography.X509Certificates" Version="4.3.2" />
-    <PackageVersion Include="System.Security.Cryptography.Xml" Version="9.0.5" />
-    <PackageVersion Include="System.Text.Json" Version="9.0.5" />
+    <PackageVersion Include="System.Security.Cryptography.Xml" Version="9.0.6" />
+    <PackageVersion Include="System.Text.Json" Version="9.0.6" />
```

Comment:
I am hoping VS has already moved to this version - otherwise we may run into problems when we insert these changes into VS (for the case where we load MS.DN.Interactive and the relevant dependencies in the devenv process as part of setting up the proxy kernels in proc). Is this something you have already tried?